code/parachain/runtime/composable/src/ibc.rs

```diff
@@ -157,13 +154,14 @@ impl ModuleRouter for Router {
 	}
 }
 
+parameter_types! {
+	pub IbcIcs20ServiceChargeZeroFee: Perbill = Perbill::from_rational(0_u32, 1000_u32 );
+}
+
 impl pallet_ibc::ics20_fee::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type ServiceCharge = IbcIcs20ServiceCharge;
+	type ServiceChargeIn = IbcIcs20ServiceChargeZeroFee;
```

Comment:
I think fee should be the same as in Picasso. Whitelisted channels will handle double charge. In case there is channel between composable and other chain, not Picasso, we should charge.

Reply:
thanks. you right.
then one more transaction should be executed on composable to add channels to whitelist.
jfyi

Reply:
Done.